docusaurus/docs/iOS/basics/overview.md

```diff
@@ -38,91 +41,18 @@ This SDK tries to keep the list of external dependencies to a minimum, these are
-
-To get started integrating Stream Chat in your iOS app, install the `StreamChatUI` dependency using one of the following dependency managers.
-
-### Install with Swift Package Manager
-
-Open your `.xcodeproj`, select the option "Add Package Dependency" in File > Swift Packages, and paste the URL: "https://github.com/getstream/stream-chat-swift".
-
-![Screenshot shows Xcode with the Add Package Dependency dialog opened and Stream Chat iOS SDK GitHub URL in the input field](../assets/spm-00.png)
-
-After pressing next, Xcode will look for the repository and automatically select the latest version tagged. Press next and Xcode will download the dependency.
-
-![Screenshot shows an Xcode screen selecting a dependency version and an Xcode screen downloading that dependency](../assets/spm-01.png)
-
-The repository contains 2 targets: StreamChat and StreamChatUI. If you'll use the UI components, select both. If you don't need the UI components, select just StreamChat.
-
-![Screenshot shows an Xcode screen with dependency targets to be selected](../assets/spm-02.png)
-
-After you press finish, it's done!
-
-:::caution
-Because StreamChat SDKs have to be distributed with its resources, the minimal Swift version requirement for this installation method is 5.3. If you need to support older Swift version, please install it using CocoaPods.
-:::
-
-
-
-
-### Install with CocoaPods
+- StreamChat the low-level client to Stream Chat API
 
-In your project's Podfile, add: `pod 'StreamChatUI', '~> 4.0.0'`. It should look similar to the snippet below.
+#### StreamChatSwiftUI
 
-```ruby
-target 'MyProject' do
-  # Comment the next line if you don't want to use dynamic frameworks
-  use_frameworks!
-
-  # Pods for MyProject
-  pod 'StreamChatUI', '~> 4.0.0'
-end
-```
-
-The StreamChatUI pod will automatically include the StreamChat dependency. If you want just the StreamChat dependency, without the UI components, add `pod 'StreamChat', '~> 4.0'` to your Podfile instead. It should look similar to the snippet below.
-
-```ruby
-target 'MyProject' do
-  # Comment the next line if you don't want to use dynamic frameworks
-  use_frameworks!
-
-  # Pods for MyProject
-  pod 'StreamChat', '~> 4.0.0'
-end
-```
-
-Now that we’ve modified our Podfile, let’s go ahead and install the project dependencies via the terminal with one simple command:
-
-```bash
-pod install --repo-update
-```
-
-The above command will generate the **MyProject.xcworkspace** file automatically.
-
-With our workspace now containing our Pods project with dependencies, as well as our original project, let’s go ahead and move over to Xcode to complete the process.
-
-### Install with Carthage
-
-Add entry to your `Cartfile`:
-
-```
-github "GetStream/stream-chat-swift" ~> 4.0.0
-```
-
-and run:
+- [Nuke](https://github.com/kean/Nuke) for loading images  
+- [NukeUI](https://github.com/kean/Nuke) for loading images  
+- [SwiftyGif](https://github.com/kirualex/SwiftyGif) for high performance GIF rendering
```

Comment:
Didn't we use Gifu for gifs? 🤔 @martinmitrevski

Reply:
Good catch! We don't use SwiftyGif, we should remove it from here (I'll remove it from the Package file as well on the other repo).